SPECS/polkit/polkit.spec

```diff
@@ -16,6 +16,7 @@ BuildRequires:     gobject-introspection
 BuildRequires:     intltool >= 0.40.0
 BuildRequires:     mozjs-devel >= 78.3.1
 BuildRequires:     pam-devel
+BuildRequires:     perl(File::Find)
```

Comment:
I didn't realize you were fixing already existing specs when we talked offline. This is a bit more suspicious, since they used to build before. Do you know what might have triggered the build failures and the additional BR now?

Reply:
Agreed this is highly suspicious. I am not sure yet of the exact cause but I'll spend time to root cause before opening this PR up for review.